src/main/webapp/WEB-INF/tags/admin/search/studentRow.tag

```diff
@@ -84,6 +84,12 @@
             <%-- Course join link --%>
             <li class="list-group-item list-group-item-info">
                 <strong>Course Join Link</strong>
+                <button type="button"
+                        onclick="mailto('${student.email}', 'Invitation to join course', '${student.name}', '${student.courseName}', 'Course Join Link', '${student.links.courseJoinLink}', '');"
```

Comment:
It is strongly preferred to add the event handler for clicks in the JS itself rather than as an `onclick` attribute.

It could go into adminSearch.es6, inside the `$(document).ready` block. Add an ID if necessary.